javascript/javascript_sdk/src/get_data.js

```diff
@@ -0,0 +1,85 @@
+const google_protobuf_timestamp_pb = require('google-protobuf/google/protobuf/timestamp_pb.js');
+
+const get_zone_data = (client) => {
+    client.listenToObjectUpdate(response => {
+      let zone_lists = response.getZonesList();
+      
+      zone_lists.forEach(zone_list => {
+          let obj = zone_list.getObject();
+          if (zone_list.getType() == 1) {
+              console.log("Enter zone(", zone_list.getId() ,") : ", obj.getId());
+          }
+          else if (zone_list.getType() == 2) {
+              console.log("Exit zone(", zone_list.getId() ,") : ", obj.getId());
+          }
+      });
+    });
+  }
+  
+const get_point_data = (client) => {
+    client.listenToPointCloudUpdate(response => {
+        const pointClouds = response.getPointsList();
+        
+        pointClouds.forEach(pointCloud =>{
+            let pointcloud_info = pointCloud.array;
+            
+            let num_points = Math.floor((pointcloud_info[2].length)/(3.0));
+            if(pointcloud_info[0] == pointMsg.PointResult.PointCloud.Type.RAW){
+            console.log("Topic {%f} no. of points - {%f}", pointcloud_info[1], num_points);
```

Comment:
Isn't the topic a string?
```suggestion
            console.log("Topic {%s} no. of points - {%f}", pointcloud_info[1], num_points);
```